src/pages/test-report/test-report.ts

```diff
@@ -21,16 +22,19 @@ export class TestReportPage extends BasePageComponent {
     public navParams: NavParams,
     public platform: Platform,
     public authenticationProvider: AuthenticationProvider,
+    public screenOrientation : ScreenOrientation,
   ) {
     super(platform, navCtrl, authenticationProvider);
   }
 
   ionViewDidEnter(): void {
     this.store$.dispatch(new TestReportViewDidEnter());
     this.deviceProvider.enableSingleAppMode();
+    this.screenOrientation.lock(this.screenOrientation.ORIENTATIONS.PORTRAIT_PRIMARY);
```

Comment:
do we need to wrap this in a `platform.ready()`?

Reply:
When testing we didn't seem to need to. My understanding of platform.ready is that it's only needed during the initial loading of the app and it's plugins, (screen orientation is in the app.module) so would be loaded when the rest of the app loads

Reply:
ah ok